src/pages/en/guides/troubleshooting.md

```diff
@@ -11,17 +11,6 @@ Astro provides several different tools to help you troubleshoot and debug your c
 
 Here are some common error messages you might see in the terminal, what they might mean, and what to do about them.
 
-### Transform failed with X error
-
-This message often appears due to a current limitation in Astro requiring your import and export statements to be at the top of your `.astro` file.
-
-**Solution**: Write your imports and exports at the top of your component script.
-
-**Status**: Current limitation; fix is being worked on.
-
-**Not sure that this is your problem?**  
-Check to see if anyone else has reported [this issue](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Transform+failed+with+*+error)!
-
```

Comment:
Removed this since it's no longer relevant, nor is the `Unexpected export` error.

https://github.com/withastro/astro/pull/4003 will provide helpful hints in-context if a user does run into an error due to `export` hoisting

Reply:
Aww... bye bye TOP NUMBER ONE ERROR.  🥲